src/Microsoft.ML.DataView/TypeManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using Microsoft.ML.Internal.DataView;
+
+namespace Microsoft.ML.Data
+{
+    public static class TypeManager
```

Comment:
>TypeManager [](start = 24, length = 11)

TypeManager is too generic of a name.  Perhaps rename to DataViewTypeManager?

Reply:
Sure.

---
In reply to: [284944816](https://github.com/dotnet/machinelearning/pull/3263#discussion_r284944816) [](ancestors = 284944816)